scikitplot/decomposition.py

```diff
@@ -160,6 +170,20 @@ def plot_pca_2d_projection(clf, X, y, title='PCA 2-D Projection', ax=None,
     for label, color in zip(classes, colors):
         ax.scatter(transformed_X[y == label, 0], transformed_X[y == label, 1],
                    alpha=0.8, lw=2, label=label, color=color)
+
+    if biplot:
+        vectors = np.transpose(clf.components_[:2, :])
+        xs = transformed_X[:, 0]
+        ys = transformed_X[:, 1]
+        for i in range(vectors.shape[0]):
+            ax.annotate("", xy=(vectors[i, 0]*max(xs), vectors[i, 1] * max(ys)),
+                        xycoords='data', xytext=(0, 0), textcoords='data',
+                        arrowprops={'arrowstyle': '-|>', 'ec': 'r'})
+
+            plt.text(vectors[i, 0] * max(xs) * 1.05, vectors[i, 1] * max(ys) * 1.05,
```

Comment:
Then here you could write `scaled_vectors[i, 0] * 1.05, scaled_vectors[i, 1] * 1.05`